src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -67,14 +67,14 @@ public class LinkTagHelper : TagHelper
         private enum Mode
         {
             /// <summary>
-            /// Rendering a fallback block if primary stylesheet fails to load. Will also do globbing if the appropriate
-            /// properties are set.
+            /// Just performing file globbing search for the href, rendering a separate &lt;link&gt; for each match.
             /// </summary>
-            Fallback,
+            GlobbedHref = 1,
             /// <summary>
-            /// Just performing file globbing search for the href, rendering a separate &lt;link&gt; for each match.
+            /// Rendering a fallback block if primary stylesheet fails to load. Will also do globbing for both the
+            /// primary and fallback hrefs if the appropriate properties are set.
             /// </summary>
-            GlobbedHref
+            Fallback = 2,
```

Comment:
- well, you coulda used `Min()` :smiling_imp: 
- why provide explicit values?  `1` is also greater than `0`